src/SmartTransactionsController.test.ts

```diff
@@ -744,4 +889,137 @@ describe('SmartTransactionsController', () => {
       expect(actual).toBe(false);
     });
   });
+
+  describe('startPollingByNetworkClientId', () => {
+    let clock: sinon.SinonFakeTimers;
+    beforeEach(() => {
+      clock = sinon.useFakeTimers();
+    });
+
+    afterEach(() => {
+      clock.restore();
+    });
+
+    it('starts and stops calling smart transactions batch status api endpoint with the correct chainId at the polling interval', async () => {
```

Comment:
would it be to tedious to use matchers and `expect(spy).toHaveBeenNthCalledWith()` instead of `mockClear`?

Reply:
sure that works!

Reply:
done here: https://github.com/MetaMask/smart-transactions-controller/pull/237/commits/2b08ccd27a429deaad6d7e318a613ab5b750626b